config/.env.sample

```diff
@@ -47,9 +47,20 @@ CANVAS_INSTANCE_URL=https://canvas-test.it.umich.edu
 CANVAS_API_CLIENT_ID=
 # Secret for the Canvas Developer Key associated with the application
 CANVAS_API_SECRET=
+# Static Canvas API token for user with admin permissions
+CANVAS_ADMIN_API_TOKEN=
+# Canvas account to which new users will be added (default: 1)
+# CANVAS_NEW_USER_ACCOUNT_ID=1
 # The Canvas custom roles IDs to use when enrolling users to sections (optional)
 # CANVAS_CUSTOM_ROLES={ "Assistant": 34, "Librarian": 21 }
 
+# Invitation API settings
+INVITATION_API_KEY=
```

Comment:
Please add back `INVITATION_API_URL=https://apps.cirrusidentity.com/console` to the `.env` file. This is the base URI from the Cirrus and this should be part of configuration. 

I was saying before i did not want the entire endpoint to the batchVSC in the configuration. `INVITATION_API_URL=https://apps.cirrusidentity.com/console/api/v1/batchInviteCsv` instead just `https://apps.cirrusidentity.com/console`.

Reply:
OK…  I had followed the code example you gave, setting them all as `const` in the code.  I didn't think moving this URL was necessary, at least not while we're only using one of Cirrus' API functions, but since you required the change, I followed your example.

I'd like for requirements and examples to be more accurate so we don't need to make changes back and forth.  These little things also add up to make the completion of these issues and this PR later and later.

---
_@pushyamig wrote in https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/247#discussion_r809084249_:

> ```
> const invitationConfig.apiURL = `https://apps.cirrusidentity.com/console`
> const cirrusAPIVersion = '/api/v1/'  # Cirrus also has `/api/v2/` but we are not using that so it make sense to me make this as constant
> const CirrusAPIEndPoint = '/batchInviteCsv`
> const URL = invitationConfig.apiURL+cirrusAPIVersion+CirrusAPIEndPoint
> ```

Reply:
Sorry, for the confusion. `https://apps.cirrusidentity.com/console` still needs to be coming form the `.env` file.

Reply:
We can chat offline for clarity purpose. Slack me if you need to.

Reply:
🏁